components/molecule/notification/src/index.js

```diff
@@ -66,10 +66,9 @@ const MoleculeNotification = ({
     setShow(showFromProps)
   }, [showFromProps])
 
-  const getAutoCloseTime = useCallback(() => {
-    const time = AUTO_CLOSE_TIME[autoCloseTiming]
-    return time
-  }, [autoCloseTiming])
+  const autoCloseTimeInSeconds = AUTO_CLOSE_TIME[autoCloseTiming]
```

Comment:
`const autoCloseTimeInSeconds = AUTO_CLOSE_TIME[autoCloseTiming]` should be enough. If not present, it will be `undefined` and thus a falsy value.

Reply:
True. But I did so to differentiate it by giving a controlled value with `null`. I add that on a personal level I like to do this to differentiate these states.
`undefined`, not initialized and perhaps not identified.
`null`, initialized and identified with a null initial value.

but maybe in this situation there are no differences, I will change it. 😆